reg.py

```diff
@@ -178,6 +189,13 @@ def multilayer_perceptron(x, weights, biases):
     print (" *********************************************** ")
 
 
-
+    # plt.plot(X_train[:, 0], sess.run(pred, feed_dict={x: X_train}), label='Fitted line')     # plotting the fitted line; y = mx + c
+    plt.plot(Y_train, sess.run(pred, feed_dict={x: X_train}), 'ro', label='Correlation of Original with Predicted data')                         # for marking points in space of labels and features
```

Comment:
You don't need to do plotting in the context of the session. Take plotting out of the session.